### PR TITLE
functional tests: add kedr with filesystem test

### PR DESCRIPTION
**Pull request by micrakow**
Signed-off-by: Michal Rakowski <michal.rakowski@intel.com>

### CODE REVIEW

File: test/functional/tests/stress/test_kedr.py [hunk 1]
```diff
@@ -208,3 +211,74 @@ def test_kedr_basic_io(module, unload_modules, install_kedr):
...
+        Kedr.start(module.value)
+
+    with TestRun.step(f"Loading CAS modules"):
+        os_utils.load_kernel_module(cas_module.CasModule.cache.value)
+
+    with TestRun.step("Starting cache"):
+        cache = casadm.start_cache(cache_part, force=True)
+
+    with TestRun.step("Adding core"):
+        core = cache.add_core(core_dev=core_part)
```
Comment (Ostrokrzew) [inline test/functional/tests/stress/test_kedr.py]: You could pass argument shortly.

File: test/functional/tests/stress/test_kedr.py [hunk 2]
```diff
@@ -208,3 +211,74 @@ def test_kedr_basic_io(module, unload_modules, install_kedr):
 
     with TestRun.step(f"Stopping kedr"):
         Kedr.stop()
+
+
+@pytest.mark.parametrize("module", cas_module.CasModule)
+@pytest.mark.require_disk("cache", DiskTypeSet([DiskType.optane, DiskType.nand]))
+@pytest.mark.require_disk("core", DiskTypeLowerThan("cache"))
+def test_kedr_basic_io_fs(module, unload_modules, install_kedr):
```
Comment (mmichal10) [inline test/functional/tests/stress/test_kedr.py]: Are you using 2nd and 3rd params? If not you can remove them in both tests

Comment (katlapinka) [reply test/functional/tests/stress/test_kedr.py]: Those are fixtures - they are executed before the test execution 😉

Comment (mmichal10) [reply test/functional/tests/stress/test_kedr.py]: Ok, I see. But now modules are unloaded in fixtures and loaded in test. Wouldn't it be more consistent to load them in fixtures?

Comment (micrakow) [reply test/functional/tests/stress/test_kedr.py]: Modules have to be unloaded for kedr to start - that's why this is (unloading, startign kedr and then loading modules) being done in test body, not in fixture itself.

**Pull request merged**